kv/db.go

```diff
@@ -123,7 +111,8 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	method = strings.TrimPrefix(method, DBPrefix)
-	if !proto.IsPublic(method) {
```

Comment:
This was the only use of `IsPublic`. Yes, `createArgsAndReply` only knows about public methods. I'll update the docs. Seems best to leave `createArgsAndReply` here next to its only usage.